Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcmVzb3VyY2Vncm91cHMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5uZXR3b3JrL2Ruc3pvbmVzL3t9L3t9L3t9/2023-07-01-preview.xml

```diff
@@ -829,6 +837,9 @@
         <arg nullable="True" type="string" var="$parameters.properties.targetResource.id" options="target-resource" group="">
           <help short="ID of an Azure resource from which the DNS resource value is taken."/>
         </arg>
+        <arg nullable="True" type="string" var="$parameters.properties.trafficManagementProfile.id" options="traffic-management-profile" stage="Experimental" group="">
+          <help short="A reference to an azure traffic manager profile resource from where the dns resource value is taken."/>
```

Comment:
Capitalize Azure Traffic Manager and DNS

Reply:
Updated.